src/onegov/org/cli.py

```diff
@@ -884,3 +886,54 @@ def mark_as_published(request: 'OrgRequest', app: 'OrgApp') -> None:
             )
 
     return mark_as_published
+
+
+@cli.command(name="delete-invisible-links")
+def delete_invisible_links() -> 'Callable[[OrgRequest, OrgApp], None]':
+    """ Deletes all the data associated with a period, including:
+    Example:
+        onegov-org --select /foo/bar delete-invisible-links
+    """
+
+    def delete_invisible_links(request: 'OrgRequest', app: 'OrgApp') -> None:
+        session = request.session
+        pages = session.query(Page).all()
+        click.echo(click.style(
+            {session.info["schema"]},
+            fg='yellow'
+        ))
+
+        invisible_links = []
+        for page in pages:
+            if getattr(page, 'text', False):  # links have no text
+                # Find links with no text, only br tags and/or whitespaces
+                soup = BeautifulSoup(page.text, 'html.parser')  # type:ignore
+                for link in soup.find_all('a'):
+                    if not any(
+                        tag.name != 'br' and (
+                            tag.name or not tag.isspace()
+                        ) for tag in link.contents
+                    ):
+                        invisible_links.append(link)
+                        click.echo(click.style(
+                            f"Page: {page.title}", fg='green'))
+                        click.echo(f"Deleting invisible link: {link}")
+                        if all(tag.name == 'br' for tag in link.contents):
+                            link.replace_with(
+                                BeautifulSoup("<br/>", "html.parser")
+                            )
+                        else:
+                            link.decompose()
+
+                    # Save the modified HTML back to page.text
+                    page.text = str(soup)  # type:ignore
```

Comment:
```suggestion
            if not isinstance(page, GeneralFileLinkExtension):
                continue
                
            for field_name in page.content_fields_containing_links_to_files:
                if text := page.content.get(field_name):
                   page.content[field_name] = remove_empty_links(text)
```
Mostly reuse the code from the Form in the Mixin and use a common utility function.